tools/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -490,7 +501,7 @@ def main():
     html_builder = HtmlBuilder(args.layout_dir)
     for input_path in args.input:
         changed_files = parse(input_path, args.output_dir, args.layout_dir,
-                              None, html_builder)
+                              None, html_builder, None)
```

Comment:
You set the default value to `None` in `parse` you do not need to explicitly set it here too.